RELEASE_NOTES.md

```diff
@@ -1,9 +1,9 @@
 Release notes
 =============
 
-### Next
-* Fixes an issue with trust anchors for eIDAS metadata resolvers not being updated when they changed at source. The VSP
-  will now update them without requiring a restart.
```

Comment:
Arguably this is a backwards-incompatible change so we should be giving it a new major version number? (Otherwise we could perhaps just make it ignore the `europeanIdentity` config?) Not that this really matters much. Bravo :clap:

Reply:
I agree, should be v3.0.0. But you set that when doing the release.

Reply:
Thank, I've reverted so the release notes are back in ### Next